### PR TITLE
ci: Change how the changelog is created

### PR DESCRIPTION
**Pull request by ieQu1**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/v5.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Please convert it to a draft if any of the following conditions are not met. Rev
 
 - [ ] Added tests for the changes
 - [ ] Changed lines covered in coverage report
-- [ ] Change log has been added to `changes/` dir
+- [ ] Change log has been added to `changes/<version>/(feat|fix)-<PR-id>.en.md` and `.zh.md` files
```
Comment (thalesmg) [inline .github/PULL_REQUEST_TEMPLATE/v5.md:8]: I think we don't need to differentiate the change type in the filename, as the commits already enforce that, and the descriptions hopefully will do as well.

Comment (thalesmg) [reply .github/PULL_REQUEST_TEMPLATE/v5.md:8]: Nevermind, now I see why we're differentiating them here.

**Review by thalesmg — DISMISSED** [review]

File: scripts/format-changelog.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    while read -r line; do
+        echo "${indent}${line}"
+        indent="  "
+    done < "${filename}"
+    echo
+}
+
+section() {
+    local section_name=$1
+    for i in "${changes_dir}"/"${section_name}"-*."${language}".md; do
```
Comment (lafirest) [inline scripts/format-changelog.sh:30]: Question:
is it necessary to sort the changes?

Comment (ieQu1) [reply scripts/format-changelog.sh:30]: I think bash globbing returns a sorted list of strings, so sorting them is unnecessary.

**Comment by ieQu1** [comment]
> I think it might be better if we can auto-call the `format-changelog.sh` in the release script

Thanks for the remark, I added it to rel/cut script

File: changes/v5.0.14-zh.md [hunk 3]
```diff
@@ -1,18 +0,0 @@
...
-  `env EMQX_BRIDGES__MQTT__XYZ__SERVER='"localhost:1883"'`。
-  此修复后，可以不使用引号，例如 `env EMQX_BRIDGES__MQTT__XYZ__SERVER='localhost:1883'`。
-
-- 通过 API 查询 `bridges` 信息时将混淆响应中的敏感数据 [#9593](https://github.com/emqx/emqx/pull/9593/)。
-
-- 使得规则引擎的行为与桥梁的指标行为更加一致：如果一个规则引擎被禁用，其指标现在会被重置 []()。
-
-## 修复
-
-- 修复了测试GCP PubSub可能泄露内存的问题，以及其JWT令牌第二次刷新失败的问题。 [#9640](https://github.com/emqx/emqx/pull/9640)
```
Comment (id) [inline changes/v5.0.14-zh.md:16]: should be 9641

changes/v5.0.14/fix-9640.zh.md --> changes/v5.0.14/fix-9641.zh.md

**Review by id — DISMISSED** [review]

**Review by id — APPROVED** [review]

**Review by qzhuyan — APPROVED** [review]

**Review by thalesmg — APPROVED** [review]

**Pull request merged**